ext/liquid_c/tokenizer.h

```diff
@@ -13,13 +13,15 @@ typedef struct token {
     enum token_type type;
     const char *str;
     long length;
+    unsigned int trim_whitespace;
```

Comment:
If you move this field after `enum token_type type;` then it would avoid unnecessary padding on x64, since right now it stores 4 bytes for the enum followed by 4 bytes of padding, then two 8 byte fields for `str` and `length`.